frontend/views/containers/proposals/AddMembers.vue

```diff
@@ -2,56 +2,81 @@
   proposal-template(
     :title='L("Add new members")'
     :rule='{ value: 5, total: 10 }'
-    :disabled='$v.form.$invalid || ($v.steps[config.steps[ephemeral.currentStep]] && $v.steps[config.steps[ephemeral.currentStep]].$invalid)'
+    :disabled='!ephemeral.isValid'
     :maxSteps='config.steps.length'
     :currentStep.sync='ephemeral.currentStep'
+    variant='addMember'
     @submit='submit'
   )
-    // | TBD in another task
-    // | https://github.com/okTurtles/group-income-simple/issues/609
-    div(v-if='currentStep === 0' key='0')
-      label.field
-        i18n.label Username
+    div(v-if='ephemeral.currentStep === 0' key='0')
+      i18n.label Username
+      label.field(
+        v-for='(member, index) in ephemeral.invitesCount'
+        :key='`member-${index}`'
+        data-test='invitee'
+      )
+        i18n.label.sr-only Username
         .input-combo
           input.input(
             type='text'
-            name='mincome'
-            required=''
-            :class='{ error: $v.form.member.$error }'
-            v-model='form.member'
-            @keyup.enter='next'
+            v-model='form.invitees[index]'
+            @keyup.enter='() => addInvitee(index)'
+            aria-required
```

Comment:
What is `aria-required`, and why are you using it instead of `required` from vuelidate like we have everywhere else?

Reply:
its different, but if we use `required` everywhere, i'll keep that one here.